Code/GraphMol/StereoGroup.h

```diff
@@ -39,20 +39,28 @@ enum class StereoGroupType {
 
  */
 class RDKIT_GRAPHMOL_EXPORT StereoGroup {
-private:
+ private:
   StereoGroupType d_grouptype;
-  std::vector<Atom *> d_atoms;
+  std::vector<Atom*> d_atoms;
 
-public:
+ public:
+  StereoGroup() : d_grouptype(StereoGroupType::STEREO_ABSOLUTE), d_atoms(0u){};
   // Takes control of atoms if possible.
-  StereoGroup(StereoGroupType grouptype, std::vector<Atom *> &&atoms);
+  StereoGroup(StereoGroupType grouptype, std::vector<Atom*>&& atoms);
   StereoGroupType getGroupType() const;
-  const std::vector<Atom *>& getAtoms() const;
+  const std::vector<Atom*>& getAtoms() const;
+  // Seems odd to have to define these, but otherwise the SWIG wrappers
+  // won't build
+  bool operator==(const StereoGroup& other) const {
```

Comment:
Looks like it is from the automagic vector equality comparison that SWIG generates. I feel like there should be a way to tell swig not to generate that operator (or SWIG should be able to deal with it using enable_if), but I can't figure that out from their documentation or code.